js/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "omi-summer-lab",
+  "version": "1.0.0",
+  "description": "none",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/IntelLedger/omi-summer-lab.git"
+  },
+  "main": "lib/index.js",
+  "scripts": {
+    "compile-protobuf": "node ./scripts/compile_protobuf.js > ./lib/protobuf/protobuf_bundle.json",
+    "test": "standard && mocha --recursive spec --compilers js:babel-register -r babel-polyfill",
+    "watch:test": "mocha -w --recursive spec --compilers js:babel-register -r babel-polyfill",
+    "postinstall": "./scripts/install_sawtooth_sdk.sh && npm run compile-protobuf",
```

Comment:
My thinking on why this failed for you is that, if you are running in vagrant, your `/project` directory is actually on the windows file system.  This probably causes some issues w.r.t. that git shallow clone.  Try cloning the repo into the user home directory, as opposed to `/project`, and it should work fine.

We'll have to defer installing it on windows hosts as part of [STL-259](https://jira.hyperledger.org/browse/STL-259)